Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM ruby:2.6-slim-stretch
+FROM ruby:2.7-slim
```

Comment:
I can't quite remember why we needed the stretch image, but I can imagine it having something to do with jemalloc. If this updates the base image far enough to need to update to libjemalloc below and everything installs and runs, then slim is good enough.

Reply:
https://github.com/zooniverse/caesar/pull/1319 hmm..not too much context. i think the introduction of -stretch was at 2.4 https://github.com/zooniverse/caesar/pull/887/files for jemalloc....from what i've read through comments and prs..i'm not getting the full context but the gist of what i understood is it was for jemalloc. 🤷‍♀️